charts/super-agent-deployment/tests/configmap_config_test.yaml

```diff
@@ -12,22 +12,49 @@ tests:
     asserts:
       - hasDocuments:
           count: 0
-  - it: super agent's config could be empty
+  - it: super agent's config always include cluster_name and namespace
     set:
+      cluster: my-cluster
       config: {}
     asserts:
       - equal:
           path: data["config.yaml"]
-          value: ""
+          value: |
+            k8s:
+              cluster_name: my-cluster
+              namespace: my-namespace
   - it: super agent's config templates
     set:
+      cluster: my-cluster
+      config:
+        content:
+          test: value
+          test2: value2
+    asserts:
+      - equal:
+          path: data["config.yaml"]
+          value: |
+            k8s:
+              cluster_name: my-cluster
+              namespace: my-namespace
+            test: value
+            test2: value2
+  - it: cluster_name and namespace from config have precedence
```

Comment:
We could add but, since there still are some unknowns (the bundle installs the super-agent or is the other way around? (..)). I wanted to rely on the common library, which is already tested, and keep this as simple as possible.

That being said, I have no problem including it if you think we should 🙂

Reply:
fair enough, no strong opinion here